src/server_manager/messages/master_messages.json

```diff
@@ -501,6 +509,30 @@
     "message": "Click 'Create'.",
     "description": "This string appears within the server setup view as an item of a list that provides instructions to configure a firewall in Google Cloud Platform. 'Create' should be translated and included untranslated in parentheses."
   },
+  "gcp_name_your_project": {
+    "message": "Name your project in the \"Project name\" field.",
```

Comment:
nit: we should be consistent with the use of single and double quotes, let's pick one and stick to it. I prefer single quotes since they don't have to be escaped, making the source strings more readable.

Reply:
Done in cae793b